haskell-indexer-translate/src/Language/Haskell/Indexer/Translate.hs

```diff
@@ -30,6 +30,7 @@ module Language.Haskell.Indexer.Translate
     --
     , XRef(..)
     , AnalysedFile(..)
+    , ModuleTick(..)
```

Comment:
Is Module a reserved name? The rest of entities don't have "Tick" suffix. Maybe remove it for consistency?